src/restview/restviewhttp.py

```diff
@@ -649,6 +656,106 @@ def launch_browser(url):
     t.start()
 
 
+class ConfigFileHandler:
+    """Creates config files and reads default options"""
+
+    # Default template for newly-created config files
+    # Contains commented out sample key/value option pairs
+    CONFIG_FILE_TEMPLATE = '''\
+    # for options description, refer to the original docs at
+    # https://github.com/mgedmin/restview?tab=readme-ov-file#synopsis
+    [DEFAULT]
```

Comment:
Maybe 
```suggestion
    [restview]
```
?

I'm not sure why I don't like DEFAULT -- partially it makes it seem like the commented options show the default values, which is not the case for at least some of them (watch).

Reply:
Done